README.md

```diff
@@ -5,15 +5,15 @@
 | [![docs-stable][docs-stable-img]][docs-stable-url] [![docs-dev][docs-dev-img]][docs-dev-url] | [![build-ci][build-ci-img]][build-ci-url] | [![codecov][codecov-img]][codecov-url] | [![release][release-img]][release-url] | [![doi][doi-img]][doi-url] |
 
 [docs-stable-img]: https://img.shields.io/badge/docs-stable-blue.svg
-[docs-stable-url]: about:blank
+[docs-stable-url]: https://juliasmoothoptimizers.github.io/BALNLSModels/stable/
 [docs-dev-img]: https://img.shields.io/badge/docs-dev-purple.svg
-[docs-dev-url]: about:blank
+[docs-dev-url]: https://juliasmoothoptimizers.github.io/BALNLSModels/dev/
 [build-ci-img]: https://github.com/JuliaSmoothOptimizers/BALNLSModels/workflows/CI/badge.svg?branch=main
-[build-ci-url]: about:blank
+[build-ci-url]: https://github.com/JuliaSmoothOptimizers/BALNLSModels/actions
 [codecov-img]: https://codecov.io/gh/JuliaSmoothOptimizers/BALNLSModels/branch/main/graph/badge.svg
-[codecov-url]: about:blank
+[codecov-url]: https://app.codecov.io/gh/JuliaSmoothOptimizers/BALNLSModels
 [release-img]: https://img.shields.io/github/v/release/JuliaSmoothOptimizers/BALNLSModels.svg?style=flat-square
-[release-url]: about:blank
+[release-url]: https://github.com/JuliaSmoothOptimizers/BALNLSModels/releases
 [doi-img]: https://zenodo.org/badge/DOI/10.5281/zenodo.5056629.svg
```

Comment:
Where does this doi come from? We need a first release before we get a doi.

Reply:
It was a copy from PDENLPModels, I only copied the SVG to not have a visual bug in the README, but it's probably not a good idea because other users might use it as the real link

Reply:
Yes. Let’s remove it for now.